client/my-sites/plugins/plugin-custom-domain-dialog/index.tsx

```diff
@@ -0,0 +1,75 @@
+import { Dialog } from '@automattic/components';
+import { useTranslate } from 'i18n-calypso';
+import { useSelector } from 'react-redux';
+import { getSelectedSiteSlug } from 'calypso/state/ui/selectors';
+import PluginIcon from '../plugin-icon/plugin-icon';
+import './style.scss';
+
+interface Props {
+	plugin: { name: string; icon: string };
+	domains: Array< { isPrimary: boolean; isSubdomain: boolean } >;
+	closeDialog: () => void;
+	isDialogVisible: boolean;
+	onProceed: () => void;
+}
+
+export const PluginCustomDomainDialog = ( {
+	plugin,
+	domains,
+	closeDialog,
+	isDialogVisible,
+	onProceed,
+}: Props ): JSX.Element => {
+	const translate = useTranslate();
+	const selectedSiteUrl = useSelector( ( state ) => getSelectedSiteSlug( state ) );
+
+	const hasNonPrimaryCustomDomain = domains.some(
+		( { isPrimary, isSubdomain } ) => ! isPrimary && ! isSubdomain
+	);
+
+	const buttons = [
+		{
+			action: 'learn-more',
+			label: translate( 'Learn more', { context: 'button label' } ),
+		},
+		{
+			action: 'manage-domains',
+			href: '/domains/manage/' + selectedSiteUrl,
+			label: translate( 'Manage domains', { context: 'button label' } ),
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **27** times:
`translate( 'Manage domains' )` *ES Score: 14*


:information_source: This string already exists without a **context**. Only add a context if the meaning of the string is very specific.